### ISSUE TITLE
Lacking Filesystem Managem Support or ZFS Support

### ISSUE THREAD

**Issue by wibed** [siderolabs/talos#7549]
tl;dr
due to limited support to provision underlying hardware either as block devices 
or to create a filesystem on top of a disk array spanning multiple block devices. 
i have to wait for zfs to be supported

long version:
Id like to run mayastor out of lack of resources to run cstor.

For this id need a openebs provisioner.
after excluding zfs ( lack of support on talos )
i am presented with 2 choices:

- https://openebs.io/docs/user-guides/localpv-hostpath
- https://github.com/openebs/device-localpv

i cannot use hostpath over an array of filesystems
i cannot pass on block devices (/dev/sdN) to let the provisioner create a filesystem for me.

i could use a single drive over the hostpath provisioner, but that defeats the purpose of using container attached storage at all.

ideally talos would have first class support for zfs and i could throw this thought problem overboard

**Comment by smira** [comment]
ZFS system extensions was merged, and it is already available for 1.5: https://github.com/siderolabs/extensions#storage

**Comment by wibed** [comment]
nope. i can forward my devices by-id

for anyone who experiences a lack of cells
https://pve.proxmox.com/wiki/Passthrough_Physical_Disk_to_Virtual_Machine_(VM)
